### DISCUSSION TITLE
Hybrid AMP and getStaticProps

### DISCUSSION THREAD

**Discussion by MickCoelho** [vercel/next.js#11188] [Help] [answered]
Is it just me or do the two not work well together? 
Calling `getStaticProps` seems to override the AMP config (only when set to hybrid).

**Comment by ijjk** [comment, marked as answer]
Hi, it looks like this is a bug and the `amp=1` query was not being honored on a hybrid AMP page with `getStaticProps`. I opened a PR here https://github.com/zeit/next.js/pull/11205 to address this

**Reply by MickCoelho** [reply]
Testing the latest canary release, I'm still running into troubles. The config isn't ignored while running `npm run dev` but `next export` doesn't export the `.amp` page

EDIT: just logged an issue about the problem https://github.com/zeit/next.js/issues/11332
